public/index.html

```diff
@@ -111,7 +111,26 @@
           >Priority
           <input class="toggle-priority" type="checkbox" />
         </label>
+        <div>Test</div>
       </form>
+    </div> -->
+
+    <!-- Summary Components -->
+    <div class="summary-wrapper">
+      <div class="summary">
+        <input id="to-do-tasks" type="checkbox" class="summary-checkbox" />
```

Comment:
the class should probably be summary__checkbox if we follow BEM naming conventions